source/includes/inkling2-guide/_overview.html.md

```diff
@@ -0,0 +1,58 @@
+
+If you're interesting in finding out more about what's going on under the hood of the Bonsai AI Engine, check out [Understand AI Engine Components][2].
+
+## Inkling: A New Language
+
+### What kind of language is Inkling?
+
+Inkling is a special-purpose language for writing AI programs. The language has a blend of declarative and functional styles. We like to describe Inkling as the world's first machine teaching language.  
+
+The current verison of Inkling replaces an earlier version. The syntax and terminology of the earlier version of Inkling is documented in the [Inkling v1 Guide][5]
+
+### What is Machine Teaching?
+
+Machine teaching is a new programming paradigm. It expresses the solution to a problem in terms of how you teach the computer to find the solution as opposed to how to calculate the solution. Machine teaching focuses on:
+
+* **What do you want the computer to know?**
+Machine teaching captures the concepts and their relationship to each other and forms a directed graph of the concepts.
+
+Imagine learning to play baseball. There are many concepts involved. It goes beyond just the rules of the game and includes things like batting stance, bat grip, hand-eye coordination, etc.
+
+* **How do you go about teaching those things?**
+Machine teaching focuses on how the problem can be taught, not how it can be calculated.
+
+Imagine teaching children to play baseball. You can't just explain the whole game to them and expect them to understand how to play. You'd break down learning into pieces and create a series of lessons that would help them learn how to play baseball.
+
+Let's say, to start, you wanted to teach them the concept of hand-eye coordination. A sequence of lessons would capture this. You might start with a whiffle ball on a tee, progress to a tee that pops the ball up, move to underhand pitches, and then overhand pitches. You could continue to build upon your hand-eye coordination concept with things like curved pitches, different styles of catching, and other exercises. These **progressive lessons** help teach one concept - hand-eye coordination.
+
+### Inkling is a Special-Purpose Language
+
+Inkling is a special-purpose language (sometimes referred to as a domain-specific language), meaning that it is intended for writing programs for a particular domain. It is not designed as a general-purpose language, which allows you to write anything you can imagine. C++, Python, Ruby, and Java are all examples of general-purpose programming languages.
+
+### Inkling is a Functional language
```

Comment:
I wonder if we should delete this section -- I suspect it causes more confusion than help...